apps/governance-e2e/src/integration/view/validators.cy.ts

```diff
@@ -0,0 +1,246 @@
+/// <reference types="cypress" />
+
+import {
+  navigateTo,
+  navigation,
+  verifyPageHeader,
+  verifyTabHighlighted,
+} from '../../support/common.functions';
+import { clickOnValidatorFromList } from '../../support/staking.functions';
+
+const guideLink = '[data-testid="staking-guide-link"]';
+const validatorTitle = '[data-testid="validator-node-title"]';
+const validatorId = '[data-testid="validator-id"]';
+const validatorPubKey = '[data-testid="validator-public-key"]';
+const ethAddressLink = '[data-testid="link"]';
+const validatorStatus = '[data-testid="validator-status"]';
+const totalStake = '[data-testid="total-stake"]';
+const pendingStake = '[data-testid="pending-stake"]';
+const stakedByOperator = '[data-testid="staked-by-operator"]';
+const stakedByDelegates = '[data-testid="staked-by-delegates"]';
+const stakeShare = '[data-testid="stake-percentage"]';
+const stakedByOperatorToolTip = "[data-testid='staked-operator-tooltip']";
```

Comment:
Nitpick: inconsistent use of single and double quotes here

Reply:
This is done. I'm pretty sure there's more inconsistencies elsewhere but I will tidy those up in another PR.